src/main/java/graphql/execution/instrumentation/tracing/TracingSupport.java

```diff
@@ -101,14 +93,18 @@ public TracingContext beginParse() {
      * @return a context to call end on
      */
     public TracingContext beginValidation() {
-        long startValidation = System.nanoTime();
+        return traceToMap(validationMap);
+    }
+
+    private TracingContext traceToMap(Map<String, Object> map) {
+        long startParse = System.nanoTime();
```

Comment:
Probably just wants to be `start` or `startPhase` or something if now generic.

Reply:
good call